frontend/online-shop/src/components/product-details/products-details.component.tsx

```diff
@@ -0,0 +1,56 @@
+import { Button, Typography } from "@mui/material";
+
+import { FC, useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { fetchProductById } from "../../services/products.service";
+
+const ProductDetail: FC = () => {
```

Comment:
You have some inconsistencies between how you named your main tsx file, the styling file and the component itself. Name all of them `ProductDetails` in the correct format of course. You currently have `product-details`, `products-details` and `ProductDetail`